source/css/sass/_mixins--spacing.scss

```diff
@@ -410,3 +410,25 @@
 @mixin small-spacing() {
   @include blg-spacing("block-end", "small");
 }
+
+@mixin wrapper() {
+  box-sizing: content-box;
+  @include constrain-width($grid-max_width, "max");
+  margin: auto;
+
+  padding-left: 7%;
```

Comment:
This is just copied from eLife. Unsure whether switching to CSS grids will change much.